src/talk/renderer/Viewer/ViewerHandlerText.vue

```diff
@@ -0,0 +1,182 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+  -->
+
+<script setup>
+import { computed, ref } from 'vue'
+import { t } from '@nextcloud/l10n'
+import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
+import NcCheckboxRadioSwitch from '@nextcloud/vue/dist/Components/NcCheckboxRadioSwitch.js'
+import NcEmptyContent from '@nextcloud/vue/dist/Components/NcEmptyContent.js'
+import NcRichText from '@nextcloud/vue/dist/Components/NcRichText.js'
+import IconCropPortrait from 'vue-material-design-icons/NoteTextOutline.vue'
```

Comment:
Maybe this icon would fit better (rotated/mirrored)? NoteText looks too squarish

```css
    transform: rotate(90deg) scaleX(-1);
```

<img width="69" alt="image" src="https://github.com/user-attachments/assets/f91217ec-016a-401b-b7f9-3a9e64bd8692">

https://pictogrammers.com/library/mdi/icon/file-chart-outline/

Reply:
Smart